src/core/rendering.js

```diff
@@ -34,6 +34,8 @@ const defaultClass = 'p5Canvas';
  * function. If <a href="#/p5/createCanvas">createCanvas()</a> is not used, the
  * window will be given a default size of 100×100 pixels.
  *
+ * Optionally, an existing canvas can be passed using a selector, ie. document.getElementById('').
```

Comment:
Here's a suggested change for the docs here to mention `setAttributes`:

```suggestion
 * Optionally, an existing canvas can be passed using a selector, ie. `document.getElementById('')`.
 * If specified, avoid using `setAttributes()` afterwards, as this will remove and recreate the existing canvas.
```